.github/workflows/deploy.yml

```diff
@@ -97,11 +97,9 @@ jobs:
           GATSBY_ALGOLIA_SEARCH_KEY: ${{ vars.GATSBY_ALGOLIA_SEARCH_KEY }}
           GATSBY_AVAILABLE_LANGUAGES: ${{ vars.GATSBY_AVAILABLE_LANGUAGES }}
           GATSBY_REALM_APP_ID: ${{ vars.GATSBY_REALM_APP_ID }}
-          GOOGLE_TRANSLATE_API_KEY: ${{ secrets.GOOGLE_TRANSLATE_API_KEY }}
           MONGODB_CONNECTION_STRING: ${{ secrets.MONGODB_CONNECTION_STRING }}
           MONGODB_REPLICA_SET: ${{ secrets.MONGODB_REPLICA_SET }}
           MONGODB_TRANSLATIONS_CONNECTION_STRING: ${{ secrets.MONGODB_TRANSLATIONS_CONNECTION_STRING }}
```

Comment:
can also remove `MONGO_TRANSLATIONS_CONNECTION_STRING` too right? here and in other files.

Reply:
In fact, that environment variable is utilized in other locations.
For example, the `MONGODB_TRANSLATIONS_CONNECTION_STRING` is loaded into `config.mongodb.translationsConnectionString` and it's used in
- Algolia index update process https://github.com/responsible-ai-collaborative/aiid/blob/4b5603012c380a6beac122717d4b6b48b2df5218/site/gatsby-site/src/scripts/algolia-update.js#L23
- In the TSNE update script https://github.com/responsible-ai-collaborative/aiid/blob/4b5603012c380a6beac122717d4b6b48b2df5218/site/gatsby-site/src/utils/updateTsne.js#L10
- In the `createDownloadIndexPage` https://github.com/responsible-ai-collaborative/aiid/blob/4b5603012c380a6beac122717d4b6b48b2df5218/site/gatsby-site/page-creators/createDownloadIndexPage.js#L14